include/exadg/incompressible_navier_stokes/spatial_discretization/turbulence_model.h

```diff
@@ -89,23 +55,35 @@ class TurbulenceModel
    */
   TurbulenceModel();
 
+  /*
+   * Destructor.
+   */
+  virtual ~TurbulenceModel();
+
   /*
    * Initialization function.
    */
   void
   initialize(dealii::MatrixFree<dim, Number> const &                matrix_free_in,
              dealii::Mapping<dim> const &                           mapping_in,
              std::shared_ptr<Operators::ViscousKernel<dim, Number>> viscous_kernel_in,
-             TurbulenceModelData const &                            data_in);
+             TurbulenceModelData const &                            turbulence_model_data_in,
+             unsigned int const                                     dof_index_velocity_in);
 
-  /*
-   *  This function calculates the turbulent viscosity for a given velocity field.
+  /**
+   * Function for *setting* the viscosity to viscosity_newtonian_limit.
```

Comment:
Is the context / naming of `viscosity_newtonian_limit` clear in this class?

Reply:
I adapted the comment, noting that we read the viscosity from the `viscous_kernel`'s data!